docs/openapi.yaml

```diff
@@ -0,0 +1,7 @@
+openapi: 3.0.0
+info:
+  title: My API
+  version: 1.0.0
+servers:
+  - url: http://localhost:5000
```

Comment:
Can we use here date from env? Look to the Swagger UI as idea for the future projects.